analysis/data_utils.py

```diff
@@ -100,17 +100,18 @@ def filter_max_time(experiment_df, max_time):
 
 # Creating "snapshots" (see README.md for definition).
 
-_DEFAULT_BENCHMARK_SAMPLE_NUM_THRESHOLD = 0.8
+_DEFAULT_BENCHMARK_SAMPLE_NUM_THRESHOLD = 1.0
 
 
 def get_benchmark_snapshot(benchmark_df,
                            threshold=_DEFAULT_BENCHMARK_SAMPLE_NUM_THRESHOLD):
-    """Finds the latest time where 80% of the trials were still running. In most
-    cases, this is the end of the experiment. In this case, we won't consider
-    the <20% of the trials that ended early for our analysis. If more than 20%
-    of the trials ended early, it's better to pick an earlier snapshot time.
-    The 80% can be overridden using the |threshold| argument. E.g., to find the
-    latest time where each trials were running, set |threshold| to 1.0.
+    """Finds the latest time where say 80% of the trials were still running (when
```

Comment:
I'm not sure the 80% part of the comment makes sense anymore.

Reply:
Done